### DISCUSSION TITLE
New storage request

### DISCUSSION THREAD

**Discussion by beatriz-mano** [biigle/core#771] [Q&A] [answered]
I have already made a new storage request. However, I was told: 'The storage request has been sent. You will be notified when it is reviewed.' I would like to know how long this type of authorization takes.

**Comment by mzur** [comment, marked as answer]
Hi, we have a new support agent for our ticket system and they are still learning how things work, so the processing of requests could take a little longer than usual. Typically, requests are processed within 24 h (excluding weekends).

**Comment by beatriz-mano** [comment]
Thank you for the help. My storage has already been approved.

I have another question: How can I measure areas in BIIGLE? And if this software has advantages over ImageJ when it comes to measuring areas? Because I've already worked with ImageJ it seems more intuitive to measure areas. in BIGLE I can't understand nor have I found tutorials with this!

**Reply by mzur** [reply]
Hi, sorry for the late reply but I was ill. If you want to measure areas, you can enable the measure tooltip in the settings of the image annotation tool: 

![image](https://github.com/biigle/core/assets/2457311/7db1011d-8335-456f-bc2c-39652f79e69d)

When this is enabled, you will see the area of an annotation in px² when you move the mouse over it. In order to show the area in m², BIIGLE needs area metadata for the images. This can be uploaded in a [metadata file](https://biigle.de/manual/tutorials/volumes/file-metadata) or determined with automatic or manual [laser point detection](https://biigle.de/manual/tutorials/laserpoints/laserpoint-detection).

To export area information, you can generate an [annotation area report](https://biigle.de/manual/tutorials/reports/reports-schema#annotation-area-report).
